inc/agent.class.php

```diff
@@ -1199,7 +1199,15 @@ protected function enrollByInvitationToken($input) {
       $pfAgent = new PluginFusioninventoryAgent();
       //var_dump($_SESSION['glpi_plugin_fusioninventory']);
       $_SESSION['glpi_fusionionventory_nolock'] = true;
+      ob_start();
       $pfCommunication->handleOCSCommunication('', $inventory, 'glpi');
+      $fiErrors = ob_get_contents();
+      if ($fiErrors) {
+         // FI print errors to sdt output and agents needs a correct response, let's save this to logs.
+         $event = __($fiErrors, 'flyvemdm');
+         $this->logInvitationEvent($invitation, $event);
+      }
```

Comment:
If FI logs something only in case of error, the we should fail the enrolment. Can you show examples of strings you catch with this code on success and on error ?

Reply:
The issue is than the output comes with one success enrollment, we detected it yesterday. Some times FI prints message like 'Unattended element in QUERY ....' before the json string showing the success enrollment.